iocore/net/quic/test/test_QUICFrame.cc

```diff
@@ -316,10 +316,41 @@ TEST_CASE("Store STREAM Frame", "[quic]")
     QUICStreamFrame stream_frame(std::move(payload), 5, 0x01000000, 0x0100000000, true);
     CHECK(stream_frame.size() == 19);
 
-    stream_frame.store(buf, &len);
+    stream_frame.store(buf, &len, 32);
     CHECK(len == 19);
     CHECK(memcmp(buf, expected, len) == 0);
   }
+
+  SECTION("split stream frame")
+  {
+    size_t len;
+    uint8_t buf1[] = {
+      0x16,                                           // 0b00010OLF (OLF=110)
+      0x01,                                           // Stream ID
+      0xc0, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x00, // Offset
+      0x0a,                                           // Data Length
+      0x01, 0x02, 0x03, 0x04, 0x05,                   // Stream Data
+      0x01, 0x02, 0x03, 0x04, 0x05,
+    };
+
+    QUICFrameUPtr frame1 = QUICFrameFactory::create(buf1, sizeof(buf1));
+    CHECK(frame1->type() == QUICFrameType::STREAM);
+    CHECK(frame1->size() == 21);
+    QUICStreamFrame *stream_frame = dynamic_cast<QUICStreamFrame *>(frame1.get());
+    CHECK(stream_frame->offset() == 0x100000000);
+    CHECK(stream_frame->stream_id() == 0x01);
+    CHECK(stream_frame->data_length() == 10);
+    QUICStreamFrame *stream_frame2 = stream_frame->split(16);
+    CHECK(stream_frame->stream_id() == 0x01);
+    CHECK(stream_frame->data_length() == 5);
+    CHECK(memcmp(stream_frame->data(), "\x01\x02\x03\x04\x05", stream_frame->data_length()) == 0);
+    CHECK(stream_frame->offset() == 0x100000000);
+
+    CHECK(stream_frame2->data_length() == 5);
+    CHECK(memcmp(stream_frame2->data(), "\x01\x02\x03\x04\x05", stream_frame2->data_length()) == 0);
```

Comment:
Can you change the data to something not 0x0102030405 to ensure we are not comparing with the first half.